### ISSUE TITLE
keyboard shortcuts shouldn't be blocked/captured by snap during development

### ISSUE THREAD

**Issue by hamidzr** [NetsBlox/NetsBlox#2039]
it would be nice to disable capturing of keyboard shortcuts for functionalities like reload (ctrl+r), or closing the tab (ctrl+w) while developing.

**Comment by brollb** [comment]
This probably should be in the client repo. 

It might be best to reuse the concept of "dev mode" already in morphic (which can be entered by shift-clicking the logo and selecting "Switch to dev mode"). This is a little inconvenient for something that is used everytime so maybe we could add a url param for automatically entering dev mode... Thoughts?

**Comment by hamidzr** [comment]
switching to dev mode doesn't seem to release the keyboard hooks. Does it do that for you? 
So we would need to first: add the requested feature to dev mode and then have a way of automatically going to dev mode. What I'm concerned about regarding reusing snap's "dev mode" is,  do we know that it won't affect how the environment is referenced through the developer console? it probably shouldn't..

**Comment by brollb** [comment]
No, it doesn't do it atm - I was just suggesting it as a way to integrate this functionality into the current environment (without adding new concepts). 

It should be mostly the same afaik. What types of changes are you worried about?

**Comment by brollb** [comment]
This actually could be exposed as an option in the settings dropdown (probably only visible on shift clicking the settings btn).

**Comment by hamidzr** [comment]
it could be but I think that would be an additional good feature. 
1. client-side toggle to disable or enable this behavior 
2. default to disabled on dev environment

**Comment by DarDoro** [comment]
There is a built-in switch to disable close warning.
Project tab started this way can be closed (Ctrl W) or refreshed (F5) without warning.

`/index.dev.html?action=example&ProjectName=AirQuality&noExitWarning`
